Library/Homebrew/utils.rb

```diff
@@ -511,14 +511,18 @@ def api_credentials
       if ENV["HOMEBREW_GITHUB_API_TOKEN"]
         ENV["HOMEBREW_GITHUB_API_TOKEN"]
       else
-        github_credentials = IO.popen("git credential-osxkeychain get", "w+") do |io|
+        github_credentials = IO.popen("git credential-osxkeychain get", "w+", :err => "/dev/null") do |io|
```

Comment:
You should use `Utils.popen_write` instead of `:err => /dev/null`. The later doesn't work for Ruby 1.8.